### PR TITLE
merged 4 PRs into one, no merge conflicts were encountered

### PR DESCRIPTION
**Pull request by murphy-codes**

### CODE REVIEW

File: node-group-project/src/components/NavBar.js [hunk 1]
```diff
@@ -15,36 +18,62 @@ const NavBar = props => {
...
       <Switch>
-        <Route exact path="/pets" component={PetTypeIndexContainer} />
+        <Route exact path="/pets" component={PetIndexContainer} />
+        <Route exact path="/pets/:name" component={PetShowContainer} />
+        <Route exact path="/past-pets" component={PetIndexContainer} />
+        <Route exact path="/classification/:classification" component={PetIndexContainer} />
+        <Route exact path="/type/:type" component={PetIndexContainer} />
+        <Route exact path="/breed/:breed" component={PetIndexContainer} />
+        <Route exact path="/employees" component={PetTypeIndexContainer} />
+        {/* <Route exact path="/grouping" render={() => <FormsContainer form={"adoption"} />} /> */}
```
Comment (DanAlberts) [inline node-group-project/src/components/NavBar.js:69]: Please remove this comment

File: node-group-project/src/components/PetIndexContainer.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  const petTiles = pets.map(pet => {
+    return (
+      <PetTile
+      key={pet.id}
+      name={pet.name}
+      breed={capFirstLetter(pet.breed)}
+      age={pet.age}
+      description={pet.description}
+      thumbnail={pet.pet_img_url}
```
Comment (DanAlberts) [inline node-group-project/src/components/PetIndexContainer.js:51]: lines 46 - 51 should be indented

File: node-group-project/src/components/NavBar.js (hunk 1 above)
Comment (murphy-codes) [reply node-group-project/src/components/NavBar.js:69]: Oops. I had meant to take that out before pushing. We were leaving that comment in as a reminder for next steps we need to take, but had not meant for it to be committed. Removed.

File: node-group-project/src/components/PetIndexContainer.js (hunk 2 above)
Comment (murphy-codes) [reply node-group-project/src/components/PetIndexContainer.js:51]: > Indented.

**Pull request merged**